css/apigee_edge.app_view.css

```diff
@@ -15,13 +15,13 @@
  * Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
  */
 
-.field {
+[class*="-app--view-mode-full"] .field {
```

Comment:
We do not want or need to use an attribute selector here. Just because it's *possible* for a theme developer to handle this themselves doesn't mean it's right to make them have to. See one of my previous comments (above), which proposes the classes to add to the Twig template.  Once implemented, this can be changed to:

```css
.apigee-app--view-mode-full x {
  ...
}
```